man/systemd.resource-control.xml

```diff
@@ -730,6 +730,9 @@
           specific to the unified hierarchy while others are specific to the legacy hierarchy. Also note that the
           kernel might support further controllers, which aren't covered here yet as delegation is either not supported
           at all for them or not defined cleanly.</para>
+
+          <para>For further details on the delegation model consult <ulink
```

Comment:
I have a weak preference for putting this nuance into systemd.resource-control rather than CGROUP_DELEGATION.md, but no biggie. I just mostly think that people might get confused because resource control doesn't take effect in the way that they necessarily intuitively expect. My interpretation is that CGROUP_DELEGATION has historically been for more about minutiae rather than things people might be expected to need to debug.

I'd say this is personal preference though, if you prefer it this way, not a problem.

Reply:
hmm, i was torn about this, but the way I see it, one is more user/admin facing the other more developer facing documentation, and I think the .control/ stuff is more something the developer needs to be aware of, and less the user/admin. But dunno, it's not clear. I tried to remedy this by adding the big fat paragraph pointing users from the user/admin facing docs to the developer docs...